### PR TITLE
Bug Bounty rules SLA, and terms&conditions

### LINKED ISSUE #389: Bug Bounty Documentation
## Summary
This issue tracks the needed changes for the upcoming Bug Bounty Program. 

Documentation requirements: 

- [x] Overview, Rules, and Scope

- [x] Service Level Agreements and time-to-remediation

- [ ] Network, task workflow, and reputation mining diagrams

- [x] Terms & Conditions 

- [x] Add a section at the top of the main README pointing to the bug bounty docs

---
As far as I can tell, no changes need to happen to the docs site configuration, but in organizing the docs, this will be put into a standalone sidebar title, with separated pages outlining the above sections (order TBD)

Accompanying this issue is also the changes to the colony.io website outlined in this issue: https://github.com/JoinColony/colonyWebsite/issues/75
(@reneegranillo and @ryanchristo are on that)

### PR DESCRIPTION
**Pull request by gichiba**
Closes #389

### CODE REVIEW

File: docs/_bug_terms.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
...unty Program is not open to any of the following persons (“Ineligible Persons”): (1) anyone who is not at least the legal age of majority, at the time of Entry, to form valid contracts in their respective country, province or state of legal residence; (2) anyone who does not satisfy any applicable legal requirements under United States and other applicable jurisdictional legal requirements for participation; or (3) in, a country, state, province or territory where the laws of the United States or local law prohibits participating or receiving a prize in the Hackathon (including, but not limited to, Brazil, Quebec, and Cuba, Sudan, Iran, North Korea, Syria, Myanmar (formerly Burma) and any other country designated by the United States Treasury's Office of Foreign Assets Control) . Employees, independent contractors under a current services agreement with Colony and members of their immediate families (defined as parents, children, siblings and spouse, regardless of where they reside and/or those living in the same household of each) are not eligible to participate in the Bug Bounty Program. Colony reserves the right to verify eligibility and to adjudicate on any dispute at any time.
```
Comment (ryanchristo) [inline docs/_bug_terms.md]: "a prize in the Hackathon"

File: docs/_bug_rules.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+_\* There are some components of the colonyNetwork repository that are not created by the Colony team, but which still could be relevant to overall security. If a bug or exploit makes use of any external libraries or submodules, it will be considered on a case-by-case basis for elegibility._
+
+## Rules
+
+
+### Submission Guidelines
+
+All bugs reported must be done through the creation of an issue in the colonyNetwork github repo, or _if the submitter wishes to disclose privately, or to remain anonymous_ by an email sent to security@colony.io.
+
+Unless there is a specific reason for a private disclosure, bugs should be submitted as issues on the colonyNetwork GitHub repository, and tagged with the 'bug' label.
```
Comment (elenadimitrova) [inline docs/_bug_rules.md]: Remove ", and tagged with the 'bug' label"

**Comment by elenadimitrova** [comment]
We will want to link to the Rinkeby release as well and mention what I said in Slack last week:

People are free to submit bounties against either
- the current testnet release (tagged on `master` branch) and deployed to Rinkeby
or
- against `develop` branch where we merge ongoing work
We will probably aim to continually release to Rinkeby on a _greenfield_ deployment roughly every 1 month / 2 sprints

**Comment by gichiba** [comment]
okie; I think this is ready to go when the checks pass :/

**Comment by gichiba** [comment]
:grimacing: ok... I think *now* it's good

**Pull request merged**